tools/ovc/openvino/tools/ovc/cli_parser.py

```diff
@@ -646,7 +646,7 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
     # Command line tool specific params
     common_group.add_argument('--output_model',
                               help='This parameter is used to name output .xml/.bin files with converted model.')
-    common_group.add_argument('--compress_to_fp16', action='store_true',
+    common_group.add_argument('--compress_to_fp16', action='store_true', default=True,
```

Comment:
Does it work for **ovc** cli tool:
ovc --input_model INPUT_MODEL --compress_to_fp16=False

Reply:
when replaced `action='store_true'` with `type=check_bool` 
```
ovc --input_model INPUT_MODEL --compress_to_fp16=False/True
```
started to work